src/Pages/Detail/Review.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from "react";
+import ReviewComment from "./ReviewComment";
+import "./Review.scss";
+
+class Review extends Component {
+  constructor() {
+    super();
+    this.state = {
+      reviewData: [
+        {
+          USER: {
+            id: "김댓글",
+            usercomment: "최고에요!",
+            date: "2020.09.03",
+            time: "22:09:26",
+          },
+
+          HOST: {
+            id: "호스트님의 답글",
+            hostcomment: "감동적인 후기 정말 감사합니다.",
+            date: "2020.09.04",
+            time: "23:08:25",
+          },
+        },
+        {
+          USER: {
+            id: "김댓글",
+            usercomment: "최고에요!",
+            date: "2020.09.03",
+            time: "22:09:26",
+          },
+
+          HOST: {
+            id: "호스트님의 답글",
+            hostcomment: "감동적인 후기 정말 감사합니다.",
+            date: "2020.09.04",
+            time: "23:08:25",
+          },
+        },
```

Comment:
이렇게가 댓글 하나인거네요!
그리고 데이터에서도 JavaScript에서는 camelCase 써주세요!
그런데 아마 백엔드에서는 아래와 같이 host가 user 안에 들어간 상태로 올 거 같습니다!
```suggestion
          USER: {
            id: "김댓글",
            usercomment: "최고에요!",
            date: "2020.09.03",
            time: "22:09:26",
            HOST: {
              id: "호스트님의 답글",
              hostcomment: "감동적인 후기 정말 감사합니다.",
              date: "2020.09.04",
              time: "23:08:25",
            },
          },
```
이유는 host가 댓글을 안달았을 수도 있어서 그럴 때는 이렇게 올 겁니다!
이렇게 해야 댓글 하나에 대한 데이터가 온전히 들어오고 있게 됩니다!
```
          USER: {
            id: "김댓글",
            usercomment: "최고에요!",
            date: "2020.09.03",
            time: "22:09:26",
            HOST: { },
          },
```
제가 하는 것처럼 프론트엔드 개발자라도 데이터가 어떤 형태로 올지 미리 예측할 수 있어야 합니다!
이 부분은 계속 개발하면서 알게 될 거지만 먼저 생각해보는 습관을 기르시면 더욱 좋을 것 같네요!

Reply:
와~~~~~네 감사합니다!